build.sh

```diff
@@ -9,8 +9,8 @@ DISTRO_DIR=src/distro/decider-spell
 mkdir -p "$DISTRO_DIR"
 
 # compile aqua file
-fluence aqua --import src/aqua --import src/aqua/decider --import src/aqua/chain --import src/aqua/fluence -i src/aqua/decider/deal_spell.aqua -o "$DISTRO_DIR" --air
-fluence aqua --import src/aqua --import src/aqua/decider --import src/aqua/chain --import src/aqua/fluence -i src/aqua/decider/main.aqua -o "$DISTRO_DIR" --air
+fluence aqua --import src/aqua --import src/aqua/decider --import src/aqua/chain --import src/aqua/fluence -i src/aqua/decider/deal_spell.aqua -o "$DISTRO_DIR/deal_spell" --air
+fluence aqua --import src/aqua --import src/aqua/decider --import src/aqua/chain --import src/aqua/fluence -i src/aqua/decider/poll.aqua -o "$DISTRO_DIR/poll" --air
```

Comment:
I put them to different dirs because of this bug 
[DXJ-403 fluence aqua --air -o DIR removes output directory](https://linear.app/fluence/issue/DXJ-403/fluence-aqua-air-o-dir-removes-output-directory)

Reply:
and yet
```
mbp16~/localdev/decider(refactor|✚1) % ./build.sh
Successfully compiled /Users/bebo/localdev/decider/src/aqua/decider/deal_spell.aqua
to /Users/bebo/localdev/decider/src/distro/decider-spell/deal_spell
Error: /Users/bebo/localdev/decider/src/aqua/decider/poll.aqua

---- Error: /Users/bebo/localdev/decider/src/aqua/chain/blocks.aqua:24:35
24     cmp <- FluenceAuroraConnector.hex_cmp(hex_a, hex_b)
                                     ^^^^^^^==============
                                     Service is found, but arrow 'hex_cmp' isn't found in scope. Did you mean any of this? 'get_env', 'poll_deals', 'blocks_diff', 'poll_deal_changed', 'latest_block_number'

---- Error: /Users/bebo/localdev/decider/src/aqua/chain/blocks.aqua:25:8
25     if cmp.success:
          ^^^=========
          Name 'cmp' isn't found in scope. Did you mean any of this? 'nil', 'less', 'hex_a', 'hex_b', 'spell_id'

---- Error: /Users/bebo/localdev/decider/src/aqua/chain/blocks.aqua:27:12
27         if cmp.ordering == -1:
              ^^^================
              Name 'cmp' isn't found in scope. Did you mean any of this? 'nil', 'less', 'hex_a', 'hex_b', 'spell_id'

---- Error: /Users/bebo/localdev/decider/src/aqua/chain/blocks.aqua:30:12
30         if cmp.ordering == 0:
              ^^^===============
              Name 'cmp' isn't found in scope. Did you mean any of this? 'nil', 'less', 'hex_a', 'hex_b', 'spell_id'

---- Error: /Users/bebo/localdev/decider/src/aqua/chain/blocks.aqua:33:12
33         if cmp.ordering == 1:
              ^^^===============
              Name 'cmp' isn't found in scope. Did you mean any of this? 'nil', 'less', 'hex_a', 'hex_b', 'spell_id'

---- Error: /Users/bebo/localdev/decider/src/aqua/chain/blocks.aqua:36:47
36         spell_log(spell_id, ["hex_cmp error", cmp.error])
                                                 ^^^========
                                                 Name 'cmp' isn't found in scope. Did you mean any of this? 'nil', 'less', 'hex_a', 'hex_b', 'spell_id'
```
for 
```
fluence --version
@fluencelabs/cli/0.5.1 darwin-x64 node-v18.16.1
```
with default depedencies.

Reply:
how about adding `fluence build` before the aqua compile ?

Reply:
@boneyard93501  what's the difference?

Reply:
I think to recompile connector service API

Reply:
yeah makes sense